_data/chains/eip155-5555.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Chain Verse Mainnet",
+  "chain": "CVERSE",
+  "network": "mainnet",
```

Comment:
```suggestion
```

Reply:
how we can specify which is our mainnet and test-net in future?